vins_estimator/CMakeLists.txt

```diff
@@ -52,6 +53,6 @@ add_executable(vins_estimator
     )
 
 
-target_link_libraries(vins_estimator ${catkin_LIBRARIES} ${OpenCV_LIBS} ${CERES_LIBRARIES}) 
+target_link_libraries(vins_estimator ${catkin_LIBRARIES} ${OpenCV_LIBS} ${CERES_LIBRARIES} spdlog ) 
```

Comment:
Any reason why we link to `spdlog::spdlog_header_only` in feature_tracker but `spdlog` here?

Reply:
Nope. Will change to header only :D